### PR TITLE
Tests use env Python and requirements.txt. Closes #13

### LINKED ISSUE #13: Use env python for tests and add requirements.txt to repo
Python scripts under `test/` reference `/usr/bin/python`. Imports in these scripts won't work if using a different Python distribution (e.g. [Anaconda](https://www.anaconda.com/what-is-anaconda/?gclid=CjwKCAjwxo3OBRBpEiwAS7X62VP6-ox50vayQGrP82cNaVCxqggxQ_LjRC0scvAF5QSf2e4sOjqrTxoCYYoQAvD_BwE)). 

Suggest switching to `#!/usr/bin/env python` to resolve the issue. 

Additionally, we should probably include a `requirements.txt` file at the top level of the repo to install Python dependencies

### PR DESCRIPTION
**Pull request by ZacharySBrown**
Updated Python scripts in test to use env Python. 
Added `requirements.txt`

Tested on:
* OS-X: El Capitan 10.11.3
* Python: Python 3.6.2 :: Continuum Analytics, Inc. (clean conda env)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+awscli==1.11.157
+boto3==1.4.7
+botocore==1.7.15
+colorama==0.3.7
```
Comment (amyschoen) [inline requirements.txt]: Are we really using all of these libraries?  I only explicitly call AWS CLI and boto3 in my test scripts.

Comment (ZacharySBrown) [reply requirements.txt]: I think the other ones are actually dependencies for `awscli`. We should be able to remove them from the `requirements.txt` though and pip will just install them when it sees `awscli`. 
Will trim them out if need be.

Comment (amyschoen) [reply requirements.txt]: Cool.  Let's trim out the extras then.  Is there any easy way to test this?  I'm also wondering if we want to get some sort of md file (maybe GettingStarted.md) to document how the install part works.

**Comment by ZacharySBrown** [comment]
Trimmed the `requirements.txt` file (now included in this PR). 

I like the idea of a `GettingStarted.md`. This PR actually came from my running through the [How to Run](https://github.com/Optum/ChaoSlingr/tree/master/test/PortChange_Generatr#how-to-run) section of the `PortChange_Generatr` test. I'll submit any other issues that come from that setup as Issues as well.

Should we just add a "GettingStarted" section to the root `README` that: 
1. Walks through installing python dependencies
2. Walks through configuring the lambda functions in your AWS account
  a. this could be done programmatically via a python script that registers the functions in `src`
3. Points folks at the test directories?

**Comment by amyschoen** [comment]
@ZacharySBrown, I didn't get the chance to pull this down and test it today.  I'll take a look tomorrow.

**Pull request merged**